src/middleware/router.coffee

```diff
@@ -151,6 +161,11 @@ sendRequestToRoutes = (ctx, routes, next) ->
               # handle mediator reponse
               responseObj = JSON.parse response.body
               ctx.mediatorResponse = responseObj
+
+              if responseObj.error?
+                ctx.autoRetry = true
```

Comment:
I don't quite understand why this is here?  I thought we would set on a channel whether or not transactions should be rerun.  This seems to overrule that?